### PR TITLE
refactor(framework):EO-1153 Refactor rxPermission

### PR DESCRIPTION
**Pull request by satyamkotakonda**
JIRA:  https://jira.rax.io/browse/EO-1153

### LGTMs
- [x] Dev LGTM
- [x] Dev+Vidyo LGTM
- [x] Design LGTM
- ~~[ ] QE LGTM (no change to functional JavaScript)~~

Before
--------

After
--------

### CODE REVIEW

**Comment by satyamkotakonda** [comment]
@CITguy  rxPermission.html file is not loading and here is screenshot.



Can you please help me here

**Comment by CITguy** [comment]
@satyamkotakonda: I've pushed a commit to the branch to fix the `templates/rxPermission.html` issue. Please be sure to pull the changes down before you continue updating the branch.

File: grunt-tasks/options/html2js.js [hunk 1]
```diff
@@ -4,7 +4,8 @@ var srcDirs = [
     'src/atoms',
     'src/components',
     'src/molecules',
-    'src/elements'
+    'src/elements',
+    'src/utilities',
```
Comment (CITguy) [inline grunt-tasks/options/html2js.js]: Added to bring in the first utility with an external template.

Comment (KalpeshBhandari) [inline grunt-tasks/options/html2js.js]: , at the end can be removed.

**Comment by satyamkotakonda** [comment]
@CITguy  Thanks very much for your support.

**Comment by satyamkotakonda** [comment]
@KalpeshBhandari  Updated changes and please review.

File: grunt-tasks/options/html2js.js (hunk 1 above)
Comment (CITguy) [inline grunt-tasks/options/html2js.js]: This is fine.

**Comment by KalpeshBhandari** [comment]
Dev LGTM

**Comment by cgtaab** [comment]
Based on screenshots, there appears to be extra spacing on the bottom of the table entry in the docs.
screenshot 2016-09-22 10 45 48

**Comment by CITguy** [comment]
@cgtaab: API docs are automatically generated and styles are out of our hands. I'm okay with the extra spacing as long as the information is correct.

**Comment by cgtaab** [comment]
Looks like the concern I raised was out of scope for this PR. Design LGTM

**Pull request merged**